metadata-ingestion/src/datahub/telemetry/telemetry.py

```diff
@@ -0,0 +1,138 @@
+import json
+import os
+import uuid
+from functools import wraps
+from pathlib import Path
+from typing import Any, Callable, Dict, Optional, TypeVar, Union
+
+import requests
+
+import datahub as datahub_package
+
+GA_VERSION = 1
+GA_TID = "UA-214428525-1"
+
+LOCAL_DIR = Path(__file__).resolve().parent
```

Comment:
What is this for?

Reply:
Oops, was using that for the config file previously. Removed!